lib/util/http_utils.dart

```diff
@@ -24,28 +24,31 @@ class HttpUtils {
 
     // this is the OAuth flow, where the authorization triggers before
     // calling the API. Leave it alone for now
-    OAuthService? oAuthService = OAuthService.values
-        .from(Utils.optionalString(api['authorization']?['oauthId']));
-    String? scope = Utils.optionalString(api['authorization']?['scope']);
-    bool forceNewTokens =
-        Utils.getBool(api['authorization']?['forceNewTokens'], fallback: false);
-    if (oAuthService != null && scope != null) {
-      OAuthServiceToken? token = await GetIt.instance<OAuthController>()
-          .authorize(context, oAuthService,
-              scope: scope, forceNewTokens: forceNewTokens);
-      if (token != null) {
-        headers['Authorization'] = 'Bearer ${token.accessToken}';
+    if (api['authorization'] != "none") {
```

Comment:
I don't see anything below that will need this. Is it inside one of the classes used?

Reply:
@vusters One part is for the bearer token, one for oauth2 and the final part is for parsing the header and body, and so on.